### PR TITLE
Replacing php-cs-fixer with pint and running pint

### PR DESCRIPTION
**Pull request by sukuasoft**
@roberto-butti 
This pull request is related to issue [#35](https://github.com/Hi-Folks/statistics/issues/35).

I removed the php-cs-fixer dependency, and the scripts that were in the composer.json file related to this dependency,
and replaced it with the pint dependency and added the script related to pint to the composer.json file. And thus resolving the replacement issue.

Create the pint configuration file and set it to the PER style pattern, and then run the pint script so that the current code changes following the PER pattern.

I also removed the .php-cs-fixer.dist.php file that served as one of the php-cs-fixer configuration files, and I also removed one of the workflow files from this same dependency.

In short, this was everything I changed in the project and carried out the tests, and they passed happily.

@aosccode

### CODE REVIEW

**Comment by roberto-butti** [comment]
Great PR @sukuasoft !
I'm going to merge it!

Roberto
